### PR TITLE
README: align README with the docs

### PR DESCRIPTION
**Pull request by mhamwala**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,42 +1,34 @@
...
+- [nodejs](incubator/nodejs/README.md) - Node.js Runtime
+- [nodejs-express](incubator/nodejs-express/README.md) - Express web framework for Node.js
+- [java-microprofile](incubator/java-microprofile/README.md) - Microprofile using Adopt OpenJDK and Maven
+- [java-spring-boot2](incubator/java-spring-boot2/README.md) - Spring Boot using IBM Java SDK and Maven
 
-Links on how can a user get started with creating a new stack
+Click here to find out more about [Appsody stacks](https://github.com/appsody/docs/blob/master/docs/stacks/stacks-overview.md).
+
+## Creating a Stack
+We are actively working to create new stacks so that more people can adopt Appsody. If you find that none of the existing stacks meet your needs please reach out to us on the [Appsody slack](https://appsody-slack.eu-gb.mybluemix.net/) or create a new GitHub issue to track the discussion.
```
Comment (helenmasters) [inline README.md]: slack -> Slack

File: README.md [hunk 2]
```diff
@@ -1,42 +1,34 @@
...
-`incubator/`:
+- `incubator/:` The stacks in the incubator folder are actively being worked on to satisfy the stable critria.
 
-`stable/`:
+- `experimental/:` Exprimental stacks are not being actively been worked on and may not fulfill the requirements of an Appsody stack. These can be used for trying out specific capabilites or proof of concept work.
 
+**Stacks currently available:**
+- [nodejs](incubator/nodejs/README.md) - Node.js Runtime
+- [nodejs-express](incubator/nodejs-express/README.md) - Express web framework for Node.js
+- [java-microprofile](incubator/java-microprofile/README.md) - Microprofile using Adopt OpenJDK and Maven
```
Comment (helenmasters) [inline README.md]: Adopt OpenJDK -> AdoptOpenJDK

Comment (mhamwala) [reply README.md]: change addressed

File: README.md [hunk 3]
```diff
@@ -1,42 +1,34 @@
...
-```
-2. Initalise Appsody stack with template
-``` bash
-appsody init <stack>/<template>
-```
-3. Launch application in development environment
-``` bash
-appsody run
-```
+This respository holds the default set of stacks avalible from the appsody cli. Stacks allow for rapid development whilst giving the stack developer the ability to control the overall applications that are created from it.
```
Comment (neeraj-laad) [inline README.md]: This respository holds the default set of stacks available from Appsody. Each stack is pre-configured with popular open source technologies to enable rapid development of quality microservice-based applications. Stacks include a base container image and project templates which act as a starting point for your application development. 

Stacks allow for rapid development whilst giving the stack developer the ability to control the overall applications that are created from it.

File: README.md [hunk 4]
```diff
@@ -1,42 +1,34 @@
...
+- `stable/:` Stable stacks meet a set of technical requirements which are yet to be defined.
 
-`incubator/`:
+- `incubator/:` The stacks in the incubator folder are actively being worked on to satisfy the stable critria.
 
-`stable/`:
+- `experimental/:` Exprimental stacks are not being actively been worked on and may not fulfill the requirements of an Appsody stack. These can be used for trying out specific capabilites or proof of concept work.
 
+**Stacks currently available:**
+- [nodejs](incubator/nodejs/README.md) - Node.js Runtime
```
Comment (neeraj-laad) [inline README.md]: can we take this list out as this will soon get outdated..

File: README.md [hunk 5]
```diff
@@ -1,42 +1,34 @@
...
+- [nodejs-express](incubator/nodejs-express/README.md) - Express web framework for Node.js
+- [java-microprofile](incubator/java-microprofile/README.md) - Microprofile using AdoptOpenJDK and Maven
+- [java-spring-boot2](incubator/java-spring-boot2/README.md) - Spring Boot using IBM Java SDK and Maven
 
-Links on how can a user get started with creating a new stack
+Click here to find out more about [Appsody stacks](https://github.com/appsody/docs/blob/master/docs/stacks/stacks-overview.md).
+
+## Creating a Stack
+We are actively working to create new stacks so that more people can adopt Appsody. If you find that none of the existing stacks meet your needs please reach out to us on the [Appsody Slack](https://appsody-slack.eu-gb.mybluemix.net/) or create a new GitHub issue to track the discussion.
```
Comment (neeraj-laad) [inline README.md]: update slack url is it not slack.appsody.dev?

Comment (neeraj-laad) [inline README.md]: new GitHub issue could be a link to take then to a new issue in this repo

**Pull request merged**